cmd/minikube/cmd/delete.go

```diff
@@ -171,6 +203,16 @@ func runDelete(cmd *cobra.Command, args []string) {
 	delCtx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
 	defer cancel()
 
+	// get image names before containers deletion
+	dockerImageNames, err := kicbaseImages(delCtx, oci.Docker)
```

Comment:
we should add these only when the user does "--purge" so we dont make delete slower for normal users

Reply:
After changing the way how we retrieve images (with `images` cmd) I moved this part of code below to condition with `purge`